docs/usage/attributes.md

```diff
@@ -84,6 +84,9 @@ Cloud providers with supported metadata endpoints and their respective attribute
 * ibmcloud-classic
   - AFTERBURN_IBMCLOUD_CLASSIC_INSTANCE_ID
   - AFTERBURN_IBMCLOUD_CLASSIC_LOCAL_HOSTNAME
+* kubevirt
+  - AFTERBURN_KUBEVIRT_HOSTNAME
+  - AFTERBURN_KUBEVIRT_INSTANCE_ID
```

Comment:
To confirm: no other metadata is available?

Reply:
optinally `instance_type` can be exposed. Added as optional attribute. Now it should be complete. :+1: